### PR TITLE
Fixes #190 - Moving from Jade templating language to Pug

### LINKED ISSUE #190: Jade to Pug?

### PR DESCRIPTION
**Pull request by willsteinmetz**
This pull request provides the transition from Jade to Pug. One change that will need to be made in the future is when the Pug 2.0.0 version comes out of beta.

### CODE REVIEW

**Comment by petecoop** [comment]
This looks good, only issue I see here is that it's using a beta package. I'd rather use stable if we can

**Comment by willsteinmetz** [comment]
There currently aren't any stable Pug packages yet. There are two super old versions and then it jumps to the 2.0.0 alphas and betas. That's the only reason I went with the beta. Hopefully it comes out of beta soon.

**Comment by petecoop** [comment]
Okay I'd like to wait on the stable release then before merging really.

**Comment by willsteinmetz** [comment]
Makes sense. I'll keep an eye out and update the PR accordingly.

**Comment by willsteinmetz** [comment]
@petecoop After a very, very, very long time, this is hopefully good to go now. Please let me know if I need to make any other changes.

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ An Expressjs generator for Yeoman, based on the express command line tool.
 - Gulp or Grunt build tools with file watching and livereload
 - .editorconfig for consistent coding styles within text editors
 - Support View engines:
-  - [Jade](http://jade-lang.com/)
+  - [Pug](http://jade-lang.com/)
```
Comment (petecoop) [inline README.md]: https://pugjs.org

File: .dependencies.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "morgan": "^1.6.1",
   "cookie-parser": "^1.3.3",
   "body-parser": "^1.13.3",
-  "jade": "^1.11.0",
+  "pug": "^2.0.0-beta5",
```
Comment (petecoop) [inline .dependencies.json]: Lets use the same version everywhere!

File: app/templates/basic-shared/package.json [hunk 3]
```diff
@@ -15,8 +15,8 @@
     "serve-favicon": "^2.3.0",
     "morgan": "^1.6.1",
     "cookie-parser": "^1.3.3",
-    "body-parser": "^1.13.3"<% if(options.viewEngine == 'jade'){ %>,
-    "jade": "^1.11.0"<% } %><% if(options.viewEngine == 'ejs'){ %>,
+    "body-parser": "^1.13.3"<% if(options.viewEngine == 'pug'){ %>,
+    "pug": "^2.0.0-beta5"<% } %><% if(options.viewEngine == 'ejs'){ %>,
```
Comment (petecoop) [inline app/templates/basic-shared/package.json]: version!

**Comment by willsteinmetz** [comment]
Argh, I'll push an update in a few minutes.

Disappointed

**Comment by willsteinmetz** [comment]
Alrighty @petecoop, I updated the noted files and actually found one other that still hadn't been updated. I did a search of the project for Jade and found no more instances so hopefully we've squashed them all now.

**Comment by petecoop** [comment]
LGTM!

**Pull request merged**